java/java-guestbook/skaffold.yaml

```diff
@@ -1,18 +1,18 @@
-apiVersion: skaffold/v1beta2
+apiVersion: skaffold/v1beta15
```

Comment:
There is another PR that updates this to v1beta16. Please sync to that after it gets in before this gets in.

Reply:
Keeping v1beta15 due to #157